src/client/Microsoft.Identity.Client/WsTrust/MexDocument.cs

```diff
@@ -225,6 +228,26 @@ private void SetPolicyEndpointAddresses(XContainer mexDocument)
             }
         }
 
+        private IEnumerable<XElement> FindElements(XContainer mexDocument, XNamespace xNamespace, string element)
+        {
+            IEnumerable<XElement> bindingElements = mexDocument.Elements().First().Elements(xNamespace + element);
```

Comment:
nit: (naming) these are no longer `biding` elements, - they can be `service`, `biding` or `policy`